adhocracy-plus/assets/scss/icons/style.css

```diff
@@ -75,3 +75,52 @@
 .font--lg {
```

Comment:
this i just saw while solving a cherry-pick conflict, but this seems to be wrong here..